src/WebJobs.Script/Config/ScriptJobHostOptions.cs

```diff
@@ -135,5 +135,11 @@ public string RootScriptPath
         /// Gets or sets the telemetry mode.
         /// </summary>
         internal TelemetryMode TelemetryMode { get; set; } = TelemetryMode.ApplicationInsights;
+
+        /// <summary>
+        /// Gets or sets a value indicating the timeout duration for the function metadata provider.
+        /// Defaults to 30 seconds.
```

Comment:
Since the default is not controlled/set here, let's remove that mention (or switch to indicate a default of `null`)